src/main/java/github/scarsz/discordsrv/modules/alerts/AlertListener.java

```diff
@@ -0,0 +1,281 @@
+                        Dynamic urlDynamic = titleDynamic.get("Url");
+                        if (urlDynamic.isPresent() && StringUtils.isNotBlank(urlDynamic.convert().intoString())) messageFormat.setTitleUrl(urlDynamic.convert().intoString());
+                    }
+
+                    Dynamic descriptionDynamic = alert.dget("Embed.Description");
+                    if (descriptionDynamic.isPresent() && StringUtils.isNotBlank(descriptionDynamic.convert().intoString())) messageFormat.setDescription(descriptionDynamic.convert().intoString());
+
+                    Dynamic fieldsDynamic = alert.dget("Embed.Fields");
+                    if (fieldsDynamic.isPresent()) {
+                        List<MessageEmbed.Field> fields = new LinkedList<>();
+                        fieldsDynamic.children().map(Object::toString).forEach(field -> {
+                            if (field.contains(";")) {
+                                String[] parts = field.split(";");
+                                if (parts.length < 2) {
+                                    return;
+                                }
+
+                                boolean inline = parts.length < 3 || Boolean.parseBoolean(parts[2]);
+                                fields.add(new MessageEmbed.Field(parts[0], parts[1], inline, true));
+                            } else {
+                                boolean inline = Boolean.parseBoolean(field);
+                                fields.add(new MessageEmbed.Field("\u200e", "\u200e", inline, true));
+                            }
+                        });
+                        messageFormat.setFields(fields);
+                    }
+
+                    Dynamic imageUrlDynamic = alert.dget("Embed.ImageUrl");
+                    if (imageUrlDynamic.isPresent() && StringUtils.isNotBlank(imageUrlDynamic.convert().intoString())) messageFormat.setImageUrl(imageUrlDynamic.convert().intoString());
+
+                    Dynamic footerDynamic = alert.dget("Embed.Footer");
+                    if (footerDynamic.isPresent()) {
+                        Dynamic textDynamic = footerDynamic.get("Text");
+                        if (textDynamic.isPresent() && StringUtils.isNotBlank(textDynamic.convert().intoString())) messageFormat.setFooterText(textDynamic.convert().intoString());
+                        Dynamic iconUrlDynamic = footerDynamic.get("IconUrl");
+                        if (iconUrlDynamic.isPresent() && StringUtils.isNotBlank(iconUrlDynamic.convert().intoString())) messageFormat.setFooterIconUrl(iconUrlDynamic.convert().intoString());
+                    }
+
+                    Dynamic timestampDynamic = alert.dget("Embed.Timestamp");
+                    if (timestampDynamic.isPresent() && timestampDynamic.as(boolean.class)) messageFormat.setTimestamp(new Date().toInstant());
```

Comment:
Differs from the other message formats, doesn't allow use of a epoch millis timestamp

Reply:
Have you tested to see if the other implementation even works for epoch timestamps? The logic to switch between the two didn’t even look like it did anything to me

Reply:
Have you tested to see if the other implementation even works for epoch timestamps? The logic to switch between the two didn’t even look like it did anything to me

Reply:
Tested, and worked